src/components/Image/Image.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 
 const Image = ({ className, webp, img, alt }) => {
   return (
-    <picture>
+    <picture className="picture">
```

Comment:
the component is Image, so we should match the classname oh lets also not add these styles, its going break images throughout the app